.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Update this one to ask users to run environment collection script?

Adding a note such as:
> # For security purposes, please check the contents of collect_env.py before running it.
would be helpful so that people don't complain later on that you had them run some code on their machine.

See https://github.com/pytorch/pytorch/blob/master/.github/ISSUE_TEMPLATE.md#system-info for reference.